docs/source/app/phasechange.rst

```diff
@@ -0,0 +1,89 @@
+~~~~~~~~~~~~~~~~~
+First, two different types of the regions, crystalline (low resistance) and amorphous (high resistance), should be distinguished. Crystalline regions are characterized by the presence of additional non-zero
+diffraction peaks on diffraction image, while the diffraction image (the frame) of amorphous regions contains only the zero-order diffraction peak.
+The presence of additional periodic diffraction peaks leads to differences in the Fourier spectrum of the crystalline frames compared to amorphous frames in the intermediate frequency range. Integration over a ring of all frames and using the result of the integration of the frames spectra over a disk as intensity for each position for sample visualization
+allows distinguishing amorphous and crystalline regions of the sample.
+
+GUI use:
+--------
+
+You can select "FFT Fourier" from the "Add Analysis" menu in the GUI to obtain crystallinity map with high contrast between crystalline and amorphous regions.
+
+..  figure:: ./images/phasechange/mode.PNG 
+
+Use the checkbox in the middle if you would like to enable/disable of masking of zero-order peak out.
+In case of enabling zero-order diffraction peak removal: use the controls on the middle to position the disk-shaped selector over the region of average frame average you'd like to mask out.
+Then, adjust the radii on the left to position the ring-shaped selector over the region of the average of frames spectra you'd like to integrate over and then click "Apply". 
+
+.. figure:: ./images/phasechange/newdatawithmasking.PNG
+
+In the left side, the resulting image will be generated. To check the quality of the detection you can use pick mode which is located under the middle image.
+In the case of correct parameters settings, crystalline regions will be brighter than amorphous.
+
+Crystalline frame:
+
+.. figure:: ./images/phasechange/newdatapickcryst.PNG
+
+Amorphous frame:
+
+.. figure:: ./images/phasechange/newdatapickam.PNG
+
+Tipps for enabling/disabling of zero-order diffraction peak removal:
+
+In case of the sharp and bright zero-order peak you will need to mask it out, as it described above (spectrum of such a peak will be a 2d sinc function which will spread all over the frames spectra)
+
+.. figure:: ./images/phasechange/newdatawithoutmasking.PNG
+
+So, the crystallinity map will not provide any contrast.
+
+But, in the case of sinc shaped zero-order peak, you do not need to remove it. The influence of it will be presented in low-frequency range and will be masked out during the integration region choosing
+.. figure:: ./images/phasechange/interf.PNG
+
+Crystalline frame:
+
+.. figure:: ./images/phasechange/pickcrystalline.PNG
+
+Amorphous frame:
+
+.. figure:: ./images/phasechange/pickam.PNG
+
+
+
+Clustering
+~~~~~~~~~~
+
+The scripting interface allows to cluster membrane, amorphous and crystalline regions with different lattice orientation in a more efficient way, taking into account regions of interests.
```

Comment:
It would be nice to have some kind of introduction here, like "to further categorize the crystalline regions [...]"